src/pupils.svg

```diff
@@ -1 +1 @@
-<svg fill="none" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M22.7 9.94L12.1 14 1.5 9.94 12.1 6l10.6 3.94z" stroke="#1D6FF5" stroke-width="2" stroke-miterlimit="10" stroke-linejoin="round"/><path d="M5.74 12v6h12.72v-6" stroke="#1D6FF5" stroke-width="2" stroke-miterlimit="10"/><path d="M2.56 11v4" stroke="#1D6FF5" stroke-width="2" stroke-miterlimit="10" stroke-linecap="round"/></svg>
\ No newline at end of file
+<svg fill="none" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M22.7 9.94L12.1 14 1.5 9.94 12.1 6l10.6 3.94z" stroke="currentColor" stroke-width="2" stroke-miterlimit="10" stroke-linejoin="round"/><path d="M5.74 12v6h12.72v-6" stroke="currentColor" stroke-width="2" stroke-miterlimit="10"/><path d="M2.56 11v4" stroke="currentColor" stroke-width="2" stroke-miterlimit="10" stroke-linecap="round"/></svg>
```

Comment:
Can we test removing all of the attributes? We should have all of these set within `Icon`, none of the `Ikonate` icons require them, they expect them to be set separately.

Reply:
I'm not very familiar with all the other attributes so I didn't know if they could safely be removed. I will have a chat with Atanas just to see if he is getting a new version that isn't blurry and then update this. Putting this into wip for now